### PR TITLE
Issue #13345: Enable examples tests for PackageNameCheckExamplesTest

### PR DESCRIPTION
**Pull request by Mostafahassen1**
Issue #13345

### CODE REVIEW

**Comment by romani** [comment]
GitHub, generate website

File: src/xdocs/checks/naming/packagename.xml [hunk 1]
```diff
@@ -58,13 +58,10 @@
...
-package com.checkstyle1.checks; // OK
-package com.checkSTYLE.checks; // OK
-package com._checkstyle.checks_; // OK
+package com.puppycrawl.tools.checkstyle.checks.naming.packagename;
+
+public class Example1 {
+}
         </source>
         <p id="Example2-config">
           An example of how to configure the check to ensure with packages start with a lowercase
```
Comment (romani) [inline src/xdocs/checks/naming/packagename.xml:67]: uppercase

File: src/xdocs/checks/naming/packagename.xml [hunk 2]
```diff
@@ -75,23 +72,17 @@ package com._checkstyle.checks_; // OK
...
-package com.checkstyle.checks; // OK
-package com.A.checkstyle.checks; // violation, name 'com.A.checkstyle' must match
-                                 // pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
-package com.checkstyle1.checks; // OK
-package com.checkSTYLE.checks; // violation, name 'com.checkSTYLE.checks' must
-                               // match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
-package com._checkstyle.checks_; // violation, name 'com._checkstyle.checks_' must match
-                                 // pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
+package com.puppycrawl.tools.checkstyle.checks.naming.packagename; // violation
+
```
Comment (romani) [inline src/xdocs/checks/naming/packagename.xml:83]:
```
package com.puppycrawl.tools.checkstyle.checks.naming.packagename; 
// violation above 'Must match pattern'
```

**Review by romani — CHANGES_REQUESTED** [review]
items to improve:

**Comment by romani** [comment]
I don't see that my review points are done

File: src/xdocs/checks/naming/packagename.xml (hunk 1 above)
Comment (Mostafahassen1) [reply src/xdocs/checks/naming/packagename.xml:67]: sorry what do you mean

Comment (romani) [reply src/xdocs/checks/naming/packagename.xml:67]: Change words to
`An example of how to configure the check to ensure with packages start with a uppercase`

File: src/xdocs/checks/naming/packagename.xml (hunk 2 above)
Comment (romani) [reply src/xdocs/checks/naming/packagename.xml:84]: please do:
`// violation above 'must match pattern'`
https://github.com/checkstyle/checkstyle/blob/8e5396f11e39711d16420cbb704c5b2da232621c/src/main/resources/com/puppycrawl/tools/checkstyle/checks/naming/messages.properties#L4

File: src/xdocs/checks/naming/packagename.xml (hunk 1 above)
Comment (romani) [reply src/xdocs/checks/naming/packagename.xml:67]: still not done

**Comment by romani** [comment]
I still don't see updates requested above as done at https://github.com/checkstyle/checkstyle/pull/16046/files

File: src/xdocs/checks/naming/packagename.xml (hunk 2 above)
Comment (Mostafahassen1) [reply src/xdocs/checks/naming/packagename.xml:84]: this is my question

**Comment by romani** [comment]
I expected update like:
```
 $ git diff
diff --git a/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/PackageNameCheckExamplesTest.java b/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/PackageNameCheckExamplesTest.java
index 2e8ca05..d805041 100644
--- a/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/PackageNameCheckExamplesTest.java
+++ b/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/PackageNameCheckExamplesTest.java
@@ -26,7 +26,6 @@ import org.junit.jupiter.api.Test;
 
 import com.puppycrawl.tools.checkstyle.AbstractExamplesModuleTestSupport;
 
-@Disabled("until https://github.com/checkstyle/checkstyle/issues/13345")
 public class PackageNameCheckExamplesTest extends AbstractExamplesModuleTestSupport {
     @Override
     protected String getPackageLocation() {
diff --git a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/packagename/Example2.java b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/packagename/Example2.java
index b9370cc..9e78815 100644
--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/packagename/Example2.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/packagename/Example2.java
@@ -10,8 +10,8 @@
 */
 
 // xdoc section -- start
-package com.puppycrawl.tools.checkstyle.checks.naming.packagename.UPPERNAME;
-// violation above 'Must match pattern'
+package com.puppycrawl.tools.checkstyle.checks.naming.packagename;
+// violation above 'must match pattern'
 
 public class Example2 {
 }
diff --git a/src/xdocs/checks/naming/packagename.xml b/src/xdocs/checks/naming/packagename.xml
index 67a75c2..d676797 100644
--- a/src/xdocs/checks/naming/packagename.xml
+++ b/src/xdocs/checks/naming/packagename.xml
@@ -64,7 +64,7 @@ public class Example1 {
 }
         </source>
         <p id="Example2-config">
-          An example of how to configure the check to ensure with packages start with a lowercase
+          An example of how to configure the check to ensure with packages start with a upper case
           letter and only contains lowercase letters or numbers is:
         </p>
         <source>
@@ -80,7 +80,7 @@ public class Example1 {
         <p id="Example2-code">Code Example:</p>
         <source>
 package com.puppycrawl.tools.checkstyle.checks.naming.packagename;
-// violation above 'Must match pattern'
+// violation above 'must match pattern'
 
 public class Example2 {
 }
diff --git a/src/xdocs/checks/naming/packagename.xml.template b/src/xdocs/checks/naming/packagename.xml.template
index 9566782..21e962a 100644
--- a/src/xdocs/checks/naming/packagename.xml.template
+++ b/src/xdocs/checks/naming/packagename.xml.template
@@ -49,7 +49,7 @@
           <param name="type" value="code"/>
         </macro>
         <p id="Example2-config">
-          An example of how to configure the check to ensure with packages start with a lowercase
+          An example of how to configure the check to ensure with packages start with a upper case
           letter and only contains lowercase letters or numbers is:
         </p>
         <macro name="example">

```

I pushed this update to your branch

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
@Mostafahassen1 , please investigate final fix and please update another Check , you should be more successful next time

**Pull request merged**